stratosphere/ncm/source/lr_redirectonlylocationresolver.cpp

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2019 Adubbz
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms and conditions of the GNU General Public License,
+ * version 2, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "impl/ncm_content_manager.hpp"
+#include "lr_redirectonlylocationresolver.hpp"
+
+namespace sts::lr {
+
+    RedirectOnlyLocationResolverInterface::~RedirectOnlyLocationResolverInterface() {
+        this->program_redirector.ClearRedirections();
```

Comment:
helper, see ContentLocationResolver comments.

Reply:
This is marked resolved but it doesn't seem to be resolved?

The helper exists but you're not using it.